requirements.txt

```diff
@@ -1,10 +1,12 @@
 autopep8==1.2.4
 click==6.6
+coverage==4.2
 flake8==2.5.4
 IPy==0.83
 line-profiler==1.0
 mccabe==0.4.0
 msgpack-python==0.4.7
+mypy-lang===0.4.6-dev-1d2a046c149e010816b6bcd5feae49e0ce3b32d2
```

Comment:
I'm not sure we should add it to the requirements. Does pip has the concept of `devDependencies` like nodeJS? I mean, `mypy` we shoulld try to keep things splitted between what is needed to:
 - build the game
 - develop the game
 - run the game
 
This to not further add to the complexity of continuous integration and the creation of a python binary .